R/zzz.R

```diff
@@ -20,6 +20,17 @@
     # `tune_args.workflow()` moved from tune to workflows
     vctrs::s3_register("generics::tune_args", "workflow", tune_args_workflow)
   }
+
+  should_register_tunable_method <- tryCatch(
```

Comment:
Can you please copy the comments from `should_register_tune_args_method` down to here as well, and update the tune version mentioned in the comment?